packages/tools/src/types/InterpolationTypes.ts

```diff
@@ -0,0 +1,18 @@
+import { Types } from '@cornerstonejs/core';
+import { Annotation } from './AnnotationTypes';
+import { InterpolationROIAnnotation } from './ToolSpecificAnnotationTypes';
+
+export type InterpolationViewportData = {
+  /** The annotation that is being updated with a change in label. */
+  annotation: InterpolationROIAnnotation;
+  /** The type of event, whether initializing the label or updating it. */
+  interpolationUID: string;
+  /** unique id of the viewport */
+  viewport: Types.IViewport;
+  sliceData: Types.ImageSliceData;
+};
+
+export type ImageInterpolationData = {
+  sliceIndex: number;
```

Comment:
since the Types.ImageSliceData is 

```
type ImageSliceData = {
  numberOfSlices: number;
  imageIndex: number;
};
```

I prefer imageIndex instead of sliceIndex just to reduce the number of concepts for people to know

Reply:
The problem is that it isn't imageIndex, nor is numberOfSlices the same as the number of images.  This has to work for non-acquisition plane volumes as well as video (which has a a frame number and a frame time, but those aren't quite image indices)